service/core/client/src/main/resources/reference.conf

```diff
@@ -28,3 +28,11 @@ lagom.circuit-breaker {
   }
 }
 #//#circuit-breaker-default
+
+#//#web-socket-client-default
+lagom.client.websocket {
+
+  #Default WebSocketClient max frame length
```

Comment:
I think we can review this comment and add some more context (e.g. Link to the documentation or copy/pasting from the docs) and a link to the server counterpart setting could be useful too.